### ISSUE TITLE
Cayo Perico missing?

### ISSUE THREAD

**Issue by JohnAllington** [FedericoDeBona/publictransport#14]
hey, we wanted to use the script on cayo perico but it looks like the nodes are missing. Unfortunately we have no idea how to insert them. It would be cool if this came as an update. :)

**Comment by FedericoDeBona** [comment]
Hi, lately I don't have much time, so if you want to try by yourself download [Open IV](https://openiv.com/) and search for vehicle nodes & nodes link files, then replace the files in [vehicle_nodes.json](https://github.com/FedericoDeBona/publictransport/blob/main/bake_data/vehicle_nodes.json) and [node_links.json](https://github.com/FedericoDeBona/publictransport/blob/main/bake_data/node_links.json) keeping the formattings. 
If you want to try and you need help, feel free to comment here

**Comment by JohnAllington** [comment]
yeah i tried it with openiv but i need to know the exactly names that i have to search for and maybe a location where to find these files.. do you know that maybe?.. if i managed to implement cayo perico i will send a pull request so everyone can use it :)

**Comment by JohnAllington** [comment]
and in which format are these files? thats what i need to know too :D

**Comment by FedericoDeBona** [comment]
You can find more information [here](https://gta.fandom.com/wiki/Paths_(GTA_V)#:~:text=common.rpf%5Cdata%5Clevels%5Cgta5%5Cpaths.xml)

**Comment by JohnAllington** [comment]
okee thanks for the information i will check that

**Comment by JohnAllington** [comment]
common.rpf\data\levels\gta5\paths.xml i found this file but it has a different format because its xml.. i dont know if the xml is useful

**Comment by FedericoDeBona** [comment]
What I've done is take the xml file and format it with a simple script (I don't have it anymore) in a json format like you see in the files in baked_data folder

**Comment by JohnAllington** [comment]
ive found the right file that contains coordinates and nodes only from cayo perico island its named "pathsheistisland.ipl" in gta directory.. but how can i bring it to the same format like the existing json files? do you know how you did that before?

**Comment by JohnAllington** [comment]
i played with it and tested it but it does not work for me.. maybe yoo are able to do it for cayo perico
